### DISCUSSION TITLE
Nextcloud AIO in an airgapped system

### DISCUSSION THREAD

**Discussion by 0Adiber** [nextcloud/all-in-one#2544] [Questions] [answered]
Is it possible to install and run AIO in an airgapped system? 

Maybe by providing all the necessary installation files and telling the container where to look for?

**Comment by szaimen** [comment]
Hi, what do you mean with airgapped system?

**Reply by 0Adiber** [reply]
Airgapped as in the host system has no internet connection.

**Comment by szaimen** [comment, marked as answer]
> Airgapped as in the host system has no internet connection.

I see. No, that is not possible
